src/main/java/com/volmit/iris/engine/object/IrisObject.java

```diff
@@ -487,23 +486,11 @@ public void setUnsigned(int x, int y, int z, Block block) {
         }
     }
 
-    public int place(int x, int z, IObjectPlacer placer, IrisObjectPlacement config, RNG rng, IrisData rdata) {
-        return place(x, -1, z, placer, config, rng, rdata);
+    public void place(int x, int yv, int z, IObjectPlacer placer, IrisObjectPlacement config, RNG rng, IrisData rdata) {
+        place(x, yv, z, placer, config, rng, null, null, rdata);
     }
 
-    public int place(int x, int z, IObjectPlacer placer, IrisObjectPlacement config, RNG rng, CarveResult c, IrisData rdata) {
-        return place(x, -1, z, placer, config, rng, null, c, rdata);
-    }
-
-    public int place(int x, int yv, int z, IObjectPlacer placer, IrisObjectPlacement config, RNG rng, IrisData rdata) {
```

Comment:
Stop cleaning up code that offers capability, I know we dont use this now, but the point of overloaded methods is to support different operations. 

One of these would have been used for cave placed objects. Revert this parameter shuffle / reduction of methods pls